### ISSUE TITLE
Can anyone reproduce the reported results? (Please do not close my issue, if it is addressed, I will close it by myself)

### ISSUE THREAD

**Issue by theodoruszq** [microsoft/transductive-vos.pytorch#20]
The previous discussion is here: https://github.com/microsoft/transductive-vos.pytorch/issues/19

I rerun with a smaller learning rate, and use the latest code. However the results are still not being matched. 

```
Per-sequence results saved in outputs//per-sequence_results-DAVIS17val.csv
--------------------------- Global results for DAVIS17val ---------------------------
 J&F-Mean    J-Mean  J-Recall   J-Decay    F-Mean  F-Recall   F-Decay
 0.663763  0.644109  0.754703  0.202716  0.683417   0.80778  0.248114
```

The paper reports 72 J&F-Mean.

If anyone reproduces it, please leave your results as a reference. And I am very curious that the Sync-BN can boost 6 points in total? Thanks in advance. 

By the way, I am not here for provocation's sake, instead it is a problem worthing to be discussed. I will not use TVOS as my baseline.

**Comment by Cloveryww** [comment]
Hi:
I think there is a bug in "transductive-vos.pytorch/modeling/network.py"`s line 31:
**self.bn256 = nn.BatchNorm2d(256)**
since using Sync-BN. Change it to:  **self.bn256 = BatchNorm2d(256)**
may be boost the performance a little. And using pytorch1.0 also improve the score a little. Although my final outcome still lower than the paper reported, but within the acceptable range.

Bisedes, I`m not sure the 3rd Sync-BN thie repo used from"https://github.com/vacancy/Synchronized-BatchNorm-PyTorch" support torch.nn.parallel.DistributedDataParallel. So, the Sync-BN problem may be the cause of performance drop. 
Hope this can help you a lot.

**Comment by theodoruszq** [comment]
@Cloveryww  Thanks for your sharing. I run this repo for several times, the performance really cannot match. Maybe it is due to the hardware difference which I think it should be acceptable. Anyway, I am trying other ways to tackle the VOS problem and it maybe meaningless to verify it.
